### PR TITLE
Howto-page: Fame and glory

### PR DESCRIPTION
**Pull request by torsava**
Added a section on how to get a badge on the Howto page.

### CODE REVIEW

**Comment by torsava** [comment]
@hroncok Looks ok?

File: portingdb/templates/howto.html [hunk 1]
```diff
@@ -51,6 +51,14 @@ <h2 id="package-it">Package it</h2>
         </p><p>
             To let us know, edit the <i>Blocks</i> field of the Bugzilla report and add the string <b>PY3PATCH-AVAILABLE</b> (you can use space as a separator). That way it will appear in our <a href="https://bugzilla.redhat.com/show_bug.cgi?id=1312032">tracking bug</a> for packages with an available patch. If need be, we'll review your patch and push it for the maintainer.
         </p>
+        <h3>Fame and Glory</h3>
+        <p>
+            After you've done the deed, you deserve your reward! Sadly, the hand of the King's daughter and half of all the kingdom has recently become unavailable (may vary upon location, ask your nearest royal family), but fret not, you can get a <i>badge</i> of honour!
+        </p><p>
+            Visit the <a href="https://fedoraproject.org/wiki/Open_Badges/Python3Log">wiki page for badges</a> and add a row with your <i>FAS username</i> if it doesn't exist yet. Inside put a link to the Bugzilla report you helped resolve as well a link to the package on PortingDB. You can get 3 badges: for your 1st, 5th and also 10th resolved package!
```
Comment (encukou) [inline portingdb/templates/howto.html]: It might be good to tell people to wait a day or two – updating the mirrors and portingdb takes some time.

**Comment by encukou** [comment]
Currently blocked by https://fedorahosted.org/fedora-badges/ticket/457

File: portingdb/templates/howto.html [hunk 2]
```diff
@@ -51,6 +51,14 @@ <h2 id="package-it">Package it</h2>
...
+        </p><p>
+            If you think we forgot to award you a badge, please ping <a href="https://fedoraproject.org/wiki/User:Churchyard">Churchyard</a> (IRC is preferred).
```
Comment (hroncok) [inline portingdb/templates/howto.html]: Make it say Miro or Miro Hrončok please.

File: portingdb/templates/howto.html (hunk 1 above)
Comment (hroncok) [inline portingdb/templates/howto.html]: Also, "wiki page for badges" seems like we are talking about some general badges-related page. I'd say "wiki page for Python 3 badges"

**Comment by hroncok** [comment]
Make the section more interesting with pics of the badges?

**Comment by torsava** [comment]
That's a great idea!

File: portingdb/templates/howto.html (hunk 2 above)
Comment (torsava) [inline portingdb/templates/howto.html]: Will do!

File: portingdb/templates/howto.html (hunk 1 above)
Comment (torsava) [inline portingdb/templates/howto.html]: I'll make it so.

**Comment by torsava** [comment]
How about now?

P.S.: I even used the correct bootstrap CSS class :)

**Comment by hroncok** [comment]
The image of the badge is old. See the current one in https://badges.fedoraproject.org/badge/parselmouth-iii

**Comment by hroncok** [comment]
Otherwise OK

**Comment by torsava** [comment]
Updated the badge!

**Pull request merged**